test/unit/parameters/common_parameter_test.rb

```diff
@@ -0,0 +1,10 @@
+require 'test_helper'
+
+class CommonParameterTest < ActiveSupport::TestCase
+  should validate_presence_of(:name)
```

Comment:
Sorry, I specifically meant the allow_value type ones below - as the name's already in PV.